tools/build_post_process.py

```diff
@@ -589,21 +599,29 @@ def store_installer_metadata(build_root, installer_path):
         "checksum": file_hash,
         "checksum_algorithm": "md5",
         "size": os.path.getsize(installer_path),
-        "installer_path": installer_path,
         "filename": os.path.basename(installer_path),
     })
-    store_build_metadata(build_root, metadata)
+    store_build_metadata(installer_root, metadata)
 
 
 def create_installer(ayon_root, build_root):
     metadata = get_build_metadata(build_root)
     ayon_version = metadata["version"]
     build_content_root = get_build_content_root(build_root, ayon_version)
+    installer_root = build_root / "installer"
+    if installer_root.exists():
+        shutil.rmtree(str(installer_root))
```

Comment:
Maybe we should add a flag for this? Imagine we are creating several versions (or platforms), you would have to  move them form the output before triggering this again, right?

Reply:
Agree, but in future PR. It would mean more changes.